packages/flutter/test/material/flat_button_test.dart

```diff
@@ -32,6 +32,33 @@ void main() {
     ]);
   });
 
+  testWidgets('Default FlatButton meets a11y contrast guidelines', (WidgetTester tester) async {
+    final SemanticsHandle handle = tester.ensureSemantics();
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: Center(
+            child: FlatButton(
+              child: const Text('FlatButton'),
+              onPressed: () { },
+            ),
+          ),
+        ),
+      ),
+    );
+
+    // Default, not disabled
```

Comment:
```suggestion
    // Default, not disabled.
```

https://www.dartlang.org/guides/language/effective-dart/documentation#do-format-comments-like-sentences

Reply:
(Here and below)